styles/scss/image-gallery.scss

```diff
@@ -273,8 +261,17 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
       padding: 2.7px;
     }
 
+    &:focus, &:hover {
```

Comment:
It's not about my needs, it's about accessibility and the needs of people who have to rely on keyboard navigation not being able to see what they've navigated to. [See WCAG 2.0 SC 2.4.7 Focus Visible](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)

I would hope you'd want to ensure your component is usable out-of-the-box by everyone and doesn't exclude an entire subset of people.

Let me know if I've misunderstood your comments.

Reply:
I wanted to make sure that it still looks good and does not affect the user experience. It is hard to code review CSS without seeing screenshots. I was under the impression that this new css was a custom one. If this was intended to fix an accessibility issue I don't have any issues but please attach some screenshots to show what you've done here.

Reply:
Again, this is intended to help improve the visibility of interactive items on the page, specifically the navigation bullets.
This screenshot is just the default state, indicating the first item is being displayed:
<img width="382" alt="Default Bullet states, with first item active" src="https://user-images.githubusercontent.com/19862719/68502736-27a40300-022f-11ea-9cd9-246c51904c0a.png">

This is intended to make it more visible to both keyboard and mouse users if they're hovering/focusing the current/active item. Because there's an active and not-active state, I went with 1.5x scaling and a used the blue from elsewhere. There are alternatives that could be considered as well. I technically didn't run a contrast check between the blue and the white, but the growth should help make it stand out, so it's not just relying on contrast.
<img width="209" alt="Active Bullet Hover/Focus State" src="https://user-images.githubusercontent.com/19862719/68502737-27a40300-022f-11ea-937b-bfce5761437e.png">
Last is hover/focus on a non-current/non-active element, the inverse of current/active hover/focus.
<img width="150" alt="Inactive Bullets Hover/Focus State" src="https://user-images.githubusercontent.com/19862719/68502738-27a40300-022f-11ea-8fd4-3caec9613250.png">